README.md

```diff
@@ -41,7 +41,7 @@ Contributers can build El-MAVEN on Windows, Ubuntu or Mac systems by following t
 `pacman --force -Sy`  
 `pacman --force -Syu`  
 `pacman --force -Su`  
-`pacman --force -Sy base-devel msys2-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-qt5 mingw64/mingw-w64-x86_64-hdf5 mingw64/mingw-w64-x86_64-netcdf mingw64/mingw-w64-x86_64-boost msys/git`  
+`pacman --force -Sy base-devel msys2-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-qt5 mingw64/mingw-w64-x86_64-hdf5 mingw64/mingw-w64-x86_64-netcdf mingw64/mingw-w64-x86_64-boost msys/git mingw-w64-x86_64-curl`
 
 - **For 32 bit**:  
```

Comment:
Let's remove this section altogether, since we do not support 32-bit systems anymore.